app/Models/DBC/Area/AreasAndZones.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models\DBC\Area;
+
+use Illuminate\Database\Eloquent\Model;
+
+/**
+ * App\Models\DBC\Area\AreasAndZones
+ *
+ * @property integer $m_ID
+ * @property integer $m_ParentAreaID
+ * @property string $m_AreaName_lang
+ * @method static \Illuminate\Database\Query\Builder|AreasAndZones whereMID($value)
+ * @method static \Illuminate\Database\Query\Builder|AreasAndZones whereMParentAreaID($value)
+ * @method static \Illuminate\Database\Query\Builder|AreasAndZones whereMAreaNameLang($value)
+ * @mixin \Eloquent
+ */
+class AreasAndZones extends Model
```

Comment:
Maybe we should defined the properties and setter and getter for better IDE autocompletion support